pkg/plugins/backendplugin/instrumentation/instrumentation.go

```diff
@@ -53,6 +53,11 @@ func instrumentPluginRequest(ctx context.Context, cfg *config.Cfg, pluginCtx *ba
 			"endpoint", endpoint,
 		}
 
+		if pluginCtx.User != nil {
+			logParams = append(logParams, "user.login", pluginCtx.User.Login)
```

Comment:
Nit. For consistency would suggest

```suggestion
			logParams = append(logParams, "uname", pluginCtx.User.Login)
```

Reply:
:thumbsup: